composer.json

```diff
@@ -2,18 +2,23 @@
   "name": "silverstripe/graphql-devtools",
   "description": "Tools to help developers building new applications on SilverStripe’s GraphQL API",
   "type": "silverstripe-vendormodule",
+  "license": "BSD-3-Clause",
   "require": {
-    "silverstripe/graphql": "*"
+    "silverstripe/graphql": "^3 || ^4",
```

Comment:
Does this actually work with graphql 3?  I'm seeing imports in other files such as `SilverStripe\GraphQL\Schema\Logger`

Reply:
The clear functionality doesn't work with v3 - but it also isn't _available_ with v3 (see yml config)

Reply:
Oh right but the rest of this module does work with graphql3? Just not clear?

Reply:
I believe it does - it at least used to and is supposed to, and I don't think anything in PR would stop it from doing so.

Reply:
It's meant to work with both of them. However, I'm thinking we should tag a v3 version and a v4 version to avoid confusion. I thought of doing it in this PR, but the travis build wanted to test both of them any way, so it was failing the build.

Reply:
Yeah, the dual support is a bit awful, though pragmatically we should just leave as is.  Perhaps do a new major at CMS 5 time that is graphql 4 only